Form/Extension/DependencyInjectionExtension.php

```diff
@@ -90,7 +90,11 @@ public function getType($name)
         $name = self::findClass($this->mappingTypes, $name);
 
         if (!isset($this->typeServiceIds[$name])) {
-            throw new InvalidArgumentException(sprintf('The field type "%s" is not registered with the service container.', $name));
+            if (class_exists($name) && in_array('Symfony\Component\Form\FormTypeInterface', class_implements($name))) {
```

Comment:
this duplicates https://github.com/symfony/form/blob/master/FormRegistry.php#L84